### PR TITLE
Prevent users from running build_proton.sh as root

### PR DESCRIPTION
**Pull request by dreamer**
Prevents users from borking their systems and wasting time wondering why schroot does not work, as per #1510

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 dist/
 build/
 contrib/
+*.swp
```
Comment (kisak-valve) [inline .gitignore:4]: Unrelated?

Comment (dreamer) [reply .gitignore:4]: I didn't think it merits its own, separate pull request.

**Comment by aeikum** [comment]
I dunno, not doing things as root is pretty much Unix 101 and this might prevent someone who does know what they're doing from doing something they want to do.
